src/models/application/preferences/mod.rs

```diff
@@ -15,17 +15,15 @@ const APP_INFO: AppInfo = AppInfo {
     author: "Jordan MacDonald",
 };
 const FILE_NAME: &str = "config.yml";
-const LINE_LENGTH_GUIDE_DEFAULT: usize = 80;
```

Comment:
Migrating these and other defaults to the preferences file while we're at it is a really nice touch. :sparkles: